src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -468,7 +505,10 @@ impl ThinPool {
             let remaining = total - used;
             if remaining <= low_water {
                 Some(if data {
-                    Bytes(IEC::Gi).sectors()
+                    match get_dirty_page_size() {
+                        Ok(dirty_page_size) => max(dirty_page_size, Bytes(IEC::Gi * 4).sectors()),
+                        Err(_) => Bytes(IEC::Gi * 5).sectors(),
```

Comment:
`IEC::Gi * 5` should be a const, and can we make it a power of 2 for my OCD's sake? :smiley_cat: either 4 or 8, don't care.

Reply:
sounds good.  I updated to : 

pub const DATA_EXPAND_SIZE: Sectors = Sectors(8 * IEC::Gi); // 8 GiB